sklearn/preprocessing/_data.py

```diff
@@ -2601,19 +2601,11 @@ def quantile_transform(X, axis=0, n_quantiles=1000,
         by np.random. Note that this is used by subsampling and smoothing
         noise.
 
-    copy : boolean, optional, (default="warn")
+    copy : boolean, optional, (default=True)
         Set to False to perform inplace transformation and avoid a copy (if the
         input is already a numpy array). If True, a copy of `X` is transformed,
         leaving the original `X` unchanged
 
```

Comment:
In some other similar places we have a ``..versionchnanged::`` directive.

Reply:
and I think it's not a bad idea to have

Reply:
yes definitely